functions.php

```diff
@@ -38,6 +38,18 @@ function insertToDB(object $db, string $title, string $author, string $release_y
     $query->execute(['title' => $title, 'author' => $author, 'release_year' => $release_year, 'genre' => $genre, 'pg_count' => $pg_count]);
 }
 
+/**
+ * @param object $db - a database object returned from the PDO at the top of index
+ * @param string $title - user entered from the form in index.php
+ *
+ * @returns void
+ */
+function updateDB(object $db, string $title): void
+{
+    $query = $db->prepare("UPDATE `books` SET `deleted` = '1' WHERE `book_title` = :title;");
```

Comment:
Risky business! Would be better to delete them via ID instead of title, just in case you ever duplicate titles in the database.